src/doom/st_lib.c

```diff
@@ -50,7 +50,10 @@ patch_t*		sttminus;
 
 void STlib_init(void)
 {
-    sttminus = (patch_t *) W_CacheLumpName(DEH_String("STTMINUS"), PU_STATIC);
+    if (W_CheckNumForName(DEH_String("STTMINUS")) > 0)
```

Comment:
`>=`?

Reply:
Fixed in https://github.com/chocolate-doom/chocolate-doom/commit/77f49e5ea5aae45fe904306a9364018c0fc23b71 .